python/lsst/dax/obscore/siav2.py

```diff
@@ -279,27 +283,7 @@ def get_band_information(
         ``bands`` key is a set of bands associated with those filters
         independent of instrument.
         """
-        matching_filters = defaultdict(set)
-        matching_bands = set()
-        with self.butler.query() as query:
-            records = query.dimension_records("physical_filter")
-            if instruments:
-                records = records.where("instrument in (INSTRUMENTS)", bind={"INSTRUMENTS": instruments})
-            for rec in records:
-                if (spec_range := self.config.spectral_ranges.get(rec.name)) is not None:
-                    spec_interval = Interval(start=spec_range[0], end=spec_range[1])
-                    assert spec_range[0] is not None  # for mypy
-                    assert spec_range[1] is not None
-                    for band_interval in band_intervals:
-                        if band_interval.overlaps(spec_interval):
-                            matching_filters[rec.instrument].add(rec.name)
-                            matching_bands.add(rec.band)
-                else:
-                    self.warnings.append(
-                        f"Ignoring physical filter {rec.name} from instrument {rec.instrument}"
-                        " since it has no defined spectral range"
-                    )
-        return {"filters": matching_filters, "bands": matching_bands}
+        raise NotImplementedError()
```

Comment:
The `Notes` in the docstring talks about default implementation, probably not relevant anymore?